.gitignore

```diff
@@ -11,3 +11,7 @@ conan-cache/
 
 # User spesific settings
 CMakeUserPresets.json
+
+# misc
+.idea/
```

Comment:
TODO: .idea is your personal work environment - this belongs into your global file (`~/.gitignore`) instead of the repository

Reply:
removed